lib/prerequisite.js

```diff
@@ -40,6 +40,18 @@ module.exports = (input, pkg, opts) => {
 			})
 		},
 		{
+			title: 'Check yarn.lock integrity',
+			enabled: () => opts.yarn,
+			task: () => execa.stdout('yarn', ['check', '--integrity'])
+				.catch(err => {
+					if (err.code === 'ENOENT') {
```

Comment:
When I use `spawn` on windows, `cp.on('close')` returns exit code `1` which means, `execa` might not handle this correctly.